packages/sprotty/src/features/projection/model.ts

```diff
@@ -19,14 +19,16 @@ import { Bounds, Dimension } from 'sprotty-protocol/lib/utils/geometry';
 import { hasOwnProperty } from 'sprotty-protocol/lib/utils/object';
 import { SChildElement, SModelRoot, SParentElement } from '../../base/model/smodel';
 import { SModelExtension } from '../../base/model/smodel-extension';
-import { BoundsAware, isBoundsAware } from '../bounds/model';
+import { transformToParentBounds } from '../../base/model/smodel-utils';
+import { isBoundsAware } from '../bounds/model';
 
 /**
  * Model elements implementing this interface can be displayed on a projection bar.
  * _Note:_ Model elements also have to be `BoundsAware` so their projections can be shown.
  */
 export interface Projectable extends SModelExtension {
-    projectionCssClasses: string[]
+    projectionCssClasses: string[],
+    projectedBounds?: Bounds,
```

Comment:
Please update the doc comment: if I understand correctly, the `projectedBounds` property will be preferred over `bounds` if present.